manifests/deploy/ocp-testing/olm/bundle/4.3/meteringconfig.crd.yaml

```diff
@@ -1548,6 +1548,29 @@ spec:
                                   properties:
                                     useInstanceCredentials:
                                       type: boolean
+                            prometheus:
+                              type: object
+                              properties:
+                                enabled:
+                                  type: boolean
```

Comment:
I think it would be helpful to add a description field for this property too.

Reply:
Done in 42681d7